Jenkinsfile

```diff
@@ -16,6 +16,7 @@ pipeline {
         stage('Setup') {
           steps {
             sh 'yarn install'
+            sh 'yarn global add rollup'
```

Comment:
Can we maybe include rollup as a dependancy and then instead run `./node_modules/bin/rollup`

I'm not a huge fan of global deps 😅 

it's very easy to get into a point where we aren't sure what version of rollup is being used in what env